scripts/soft-reboot

```diff
@@ -168,10 +167,10 @@ if [ -x ${WATCHDOG_UTIL} ]; then
     ${WATCHDOG_UTIL} arm
 fi
 
-# Run platform specific reboot plugin
-if [ -x ${DEVPATH}/${PLATFORM}/${PLATFORM_PLUGIN} ]; then
-    debug "Running ${PLATFORM} specific plugin..."
-    ${DEVPATH}/${PLATFORM}/${PLATFORM_PLUGIN}
+# Run platform specific ssd firmware update plugin
+if [ -x ${DEVPATH}/${PLATFORM}/${SSD_FW_UPDATE} ]; then
+    debug "Updating ssd fw for ${REBOOT_TYPE}"
+    ${DEVPATH}/${PLATFORM}/${SSD_FW_UPDATE} ${REBOOT_TYPE}
```

Comment:
Why do we replace a generic solution with a specific solution?

Reply:
I will put it back.